### ISSUE TITLE
abnormal prepulse inhibition parents

### ISSUE THREAD

**Issue by sbello** [obophenotype/mammalian-phenotype-ontology#3061]
I have another request for you. MP:0003088 , “abnormal prepulse inhibition” is a child of “nervous system phenotype” and the IMPC behavioural working group thought that it should be a child of abnormal behaviour as well. Would it be possible to make it a child of abnormal behaviour too?

Cheers,
Piia

I'm not sure why this was placed where it is, most of the annotations refer to behavior related phenotypes.

**Comment by anna-anagnostop** [comment]
in GO, prepulse inhibition is a startle response which is a neuromuscular process which in turn is a nervous system process; hence the placement as a child of nervous system phenotype.

**Comment by sbello** [comment]
Yes, but the publications talk about it as a behavioral assay and many of the notes are about acoustic startle. I think the confusion is that it is a neurological process with a behavioral read out.
